.travis/install-kmip-kms-noobaa.sh

```diff
@@ -4,7 +4,7 @@ set -o errexit
 container=$(docker run -d \
     -e "DOCKER_STEPCA_INIT_NAME=The Authority" \
     -e "DOCKER_STEPCA_INIT_DNS_NAMES=localhost,$(hostname -f)" \
-    smallstep/step-ca)
+    smallstep/step-ca:sha256-eff4b06d122afbdc8fd3a89c20680a3a35782c41118ad8ad3a290d3f03e7647d.sig)
```

Comment:
```suggestion
    smallstep/step-ca:0.23.1)
```

Reply:
I'll try `0.23.0`. I suspect it started with `0.23.1-rc.1`